storybook/src/elements/nuxeo-pagination-controls/nuxeo-pagination-controls.stories.js

```diff
@@ -0,0 +1,11 @@
+import { html } from 'lit-html';
+import { storiesOf } from '@storybook/polymer';
+import '@nuxeo/nuxeo-ui-elements/nuxeo-pagination-controls.js';
+
+storiesOf('UI/nuxeo-pagination-controls', module).add(
+  'Nuxeo pagination',
+  () => html`
+    <style></style>
+    <nuxeo-pagination-controls page="1" number-of-pages="10"> </nuxeo-pagination-controls>
```

Comment:
Could you please add a knob to define the number of pages? It would be very useful for validation, testing, etc.